src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -904,10 +912,15 @@ await Parallel.ForEachAsync(location.Value.EnumerateFiles(), async (file, token)
                         if (inState.TryGetValue(gamePath, out var entry))
                         {
                             var fileInfo = file.FileInfo;
-
-                            // If the files don't match, update the entry
-                            if (fileInfo.LastWriteTimeUtc > entry.LastModified || fileInfo.Size != entry.Size)
+                            
+                            // Dates in computers are so utterly borked it's not even funny. Turns out here there's an issue, a DateTimeOffset
+                            // when run through DateTime and/or an ulong conversion will lose a very small amount of precision, about 150μs worth. 
+                            // But this precision loss is negated if everything just uses a long. So convert both sides to a long and compare.
+                            // If you ever want to try and fix this, simply subtract one from the other, and you'll see a timespan that does not
+                            // equal zero. This is the precision loss, I (halgari) am not sure how else to fix it.
+                            if (fileInfo.LastWriteTimeUtc.Date.ToFileTimeUtc() > entry.LastModified.Date.ToFileTimeUtc() || fileInfo.Size != entry.Size)
```

Comment:
I'm not sure I follow why `.Date` is used here, was it to fix the accuracy issue? 
From reading the docs, `.Date` returns a `DateTime` of just the date information and time set at midnight, so this just checks whether there is a difference of more than a day?

Reply:
I tried date, and it's still off as well. I think there's strange culture/timezone stuff in all these cases, so the method that we know always works is just to use the underlying values

Reply:
Current code uses `.Date`, which cuts off all the time data to midnight. 
So I think the current code is wrong in the sense that it checks if the created Date is greater, with 1day resolution, rather than comparing TimeStamps.

Reply:
The issue is deeper than this and will require a DB migration, so I'm pulling that change out, and will get a new PR up with this change soon